test/e2e/specs/all-features-confirmation/record/presentation.spec.js

```diff
@@ -163,4 +170,92 @@ describe('View existing record,', function() {
+        });
+        it('Table of contents should be collapsed by default', function(){
+            var recordSidePan = chaisePage.recordPage.getElementById('record-side-pan');
+            recordSidePan.isDisplayed().then(function (bool) {
+                expect(bool).toBeFalsy();
+            });
+        });
+
+        it('Table of contents should match all related table', function(){
+            var  allPan = element.all(by.xpath("//div[contains(@id,'recordSidePan')]"));
+            chaisePage.waitForElement(fiddlerBtn).then(function(){
+              return fiddlerBtn.click();
+            }).then(function(){
+              expect(allPan.count()).toBe(testParams.sidePanelTest.tocCount, "Table of contents count did not match!");
+            }).catch( function(err) {
+               console.log(err);
+           });
+        });
+
+        it('On click of Related table name in TOC, page should move to the contents and open the table details', function(){
+            var rtTableHeading = chaisePage.recordPage.getRelatedTableHeading(testParams.sidePanelTest.tableToShow),
+                lastRow = element(by.id('rt-heading-Categories_5')).all(by.tagName('tr')).get(5);
+
+            recSidePan.getAttribute('class').then(function(className) {
+                return recSidePan.click();
+            }).then (function (){
+              // related table should be visible
+                rtTableHeading.isDisplayed().then(function (bool) {
+                  expect(bool).toBeTruthy();
+                });
+                chaisePage.waitForElement(lastRow);
+                return rtTableHeading.getAttribute("class");
+            }).then (function(className) {
+                expect(className).toContain("panel-open", "Related table panel is not open when clicked through TOC.");
+            }).catch( function(err) {
+                console.log(err);
+            });
+        });
+
+        it('Record count should match for pan and related table content', function(){
+            var rtRow = browser.executeScript("return $('#rt-Categories_5 tbody tr')"),
+                rtPanRecordCount = element(by.id('recordSidePan-heading-5')).all(by.tagName('span')).get(1),
+                rtCount;
+
+            rtRow.then(function(rt){
+              rtCount = rt;
+              return rtPanRecordCount.getText();
+            }).then(function(panRecordCount){
+               expect(parseInt(panRecordCount.slice(1, 2))).toBe(rtCount.length, "Record count did not match with side pan");
+             }).catch( function(err) {
+                console.log(err);
+            });
+        });
+
+        it('Side panel should hide/show by clicking pull button', function(){
+            var rtTableHeading = chaisePage.recordPage.getRelatedTableHeading(testParams.sidePanelTest.tableToShow),
+                recPan =  chaisePage.recordPage.getElementById('record-side-pan');
+
+            recSidePan.getAttribute('class').then(function(className) {
```

Comment:
What is the point of this function? You are not using `className` as part of it's callback.